docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+Decommission SecureDrop
+=======================
+
+The following steps will guide you through the decommissioning of your
+SecureDrop instance.
+
+#. **Put a notice in advance on your landing page to inform sources that your
```

Comment:
s/be retiring/be retired/
An instance can be retired, but it can't retire - if you see what I mean.

Reply:
fixed in f38127f